<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphEditorDockWidget</class>
 <widget class="QDockWidget" name="GraphEditorDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphEditorDockWidget</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="PropertyGridWidget" name="propertyGridWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="graphEditorLayout"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="viewportLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionTest1">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/Toolbar/File/OpenApp24.png</normaloff>:/Toolbar/File/OpenApp24.png</iconset>
   </property>
   <property name="text">
    <string>Test1</string>
   </property>
  </action>
  <action name="actionTest1_2">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/Toolbar/File/SaveScene24.png</normaloff>:/Toolbar/File/SaveScene24.png</iconset>
   </property>
   <property name="text">
    <string>Test1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertyGridWidget</class>
   <extends>QWidget</extends>
   <header>PropertyGrid/propertygridwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
